### PR TITLE
Enable CI

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Enable CI for running tests and deploying.

~Note: PR has been set to use `py3` as a base to simplify diffing, but should actually be merged to master after #1 gets merged.~

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,94 +1,22 @@
...
-"""
-
 from setuptools import setup
 
 
 setup(
     name='celery-unique',
-    version='1.0.0',
     url='https://github.com/sgrepo/celery-unique',
     author='Tyler Hendrickson, Shiftgig Inc',
```
Comment (TylerHendrickson) [inline setup.py:18]: You should add yourself here and to the next line at some point

Edit: As-discussed, we should move to using a CONTRIBUTORS/AUTHORS file instead

File: README.rst [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+celery-unique
+=============
+
 Adds argument-based unique constraints to Celery tasks
 
-## Overview
+Overview
+--------
 Configuring unique constraints on Celery requires the following:
 - A modified Task base class for Celery (provided by this package)
```
Comment (TylerHendrickson) [inline README.rst]: List doesn't appear to be rendering in rst

File: README.rst [hunk 3]
```diff
@@ -1,17 +1,20 @@
-# celery-unique
+celery-unique
```
Comment (TylerHendrickson) [inline README.rst]: As-discussed, perhaps we can use `pandoc` to assist with automatically converting `README.md` to `README.rst` as a deployment step.  If that seems feasible, we should change this file back to markdown.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+script:
+  - tox
+
+before_deploy:
+  - pip install wheel
+  - python setup.py sdist bdist_wheel
+
+deploy:
+  - provider: pypi
+    user: hobarrera
```
Comment (TylerHendrickson) [inline .travis.yml]: Will provide a Shiftgig user and credentials for you

**Review by TylerHendrickson — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 4 above)
Comment (WhyNotHugo) [reply .travis.yml]: Ping?

**Comment by WhyNotHugo** [comment]
I think I screwed up since I hadn't pushed my latest working branch for your last review, and I'd rewritten some commits locally (thought I hadn't pushed them, sorry).

Anyway, this is ready for a second round or review.

File: setup.py (hunk 1 above)
Comment (WhyNotHugo) [reply setup.py:18]: See #3

**Review by TylerHendrickson — APPROVED** [review]

**Pull request merged**